examples/sample-app/cleanup.sh

```diff
@@ -3,8 +3,13 @@
 echo "Killing openshift all-in-one server ..."
 pkill -x openshift
 
+echo "Stopping all k8s docker containers on host ..."
+docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
```

Comment:
@bparees ok, got it, this command will only instread of docker ps. I will fix it.